src/js/edu-benefits/5490/config/form.js

```diff
@@ -241,7 +239,7 @@ const formConfig = {
                 }
               }),
               'ui:options': {
-                hideIf: (fieldData) => fieldData.relationship !== 'spouse'
+                hideIf: (pageData, allData) => _.get('applicantInformation.relationship', allData) !== 'spouse'
```

Comment:
Just as long as we're talking about naming conventions, should it be `formData` or `allData`? It should be consistent either way.